src/PowerSystems.jl

```diff
@@ -321,18 +323,19 @@ export has_supplemental_attributes
 export iterate_supplemental_attributes
 export get_time_series
 export get_time_series_array
-export get_time_series_resolution
+export list_time_series_resolutions
 export get_time_series_timestamps
 export get_time_series_values
-export get_time_series_names
 export get_time_series_counts
 export get_scenario_count
 export get_percentiles
 export get_next_time_series_array!
 export get_next_time
 export get_horizon
 export get_forecast_initial_times
-export get_forecast_total_period
+export list_time_series_info
```

Comment:
This is a new way of finding time series information programmatically without actually reading the data from the .h5. (I don't love the name, but couldn't think of anything else.) There is also `list_time_series_metadata`, which I intentionally did not re-export here. Similar to the prior implementation where you could view the metadata inside `component.time_series_metadata`, it is not helpful to users because it has `*Metadata` struct names.